src/api/custom-resolvers/queries/get-ordnance-survey-addresses/index.test.ts

```diff
@@ -95,4 +81,18 @@ describe('getOrdnanceSurveyAddresses', () => {
       });
     });
   });
+
+  describe('when ordnanceSurvey API throws an error', () => {
+    beforeEach(() => {
+      ordnanceSurvey.get = mockSpyPromiseRejection;
+    });
+
+    it('should return object containing success=false and apiError=true', async () => {
+      const response = await getOrdnanceSurveyAddress({}, { postcode, houseNameOrNumber });
+
+      const expected = { success: false, apiError: true };
+
+      expect(response).toEqual(expected);
```

Comment:
Should this not be `rejects.toThrow(...)`, also can we add an assertion to check the error message text please?

Reply:
Good spot - have updated to throw an error in the catch handler.

Reply:
Thanks